### PR TITLE
Pare down related to domain

### PR DESCRIPTION
**Pull request by dannyroberts**
Don't merge!

some tech debt related to https://github.com/dimagi/commcare-hq/pull/8389, no specific timeline necessary.

### CODE REVIEW

File: corehq/couchapps/all_docs/views/by_domain_doc_type/map.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function (doc) {
```
Comment (dimagimon) [inline corehq/couchapps/all_docs/views/by_domain_doc_type/map.js]: Missing name in function declaration.

**Comment by dimagimon** [comment]
There are 241 errors:

* corehq/apps/domain/dbaccessors.py, line 20 - F821 undefined name 'length'
* corehq/apps/reminders/tests/test_everything.py, line 13 - E501 line too long (122 > 115 characters)
* corehq/apps/reminders/tests/test_everything.py, line 46 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 81 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 83 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 83 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 84 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 84 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 84 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 84 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 85 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 85 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 85 - E231 missing whitespace after ':'
* corehq/apps/reminders/tests/test_everything.py, line 85 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 86 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 86 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 204 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 206 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 206 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 207 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 207 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 207 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 207 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 208 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 208 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 208 - E231 missing whitespace after ':'
* corehq/apps/reminders/tests/test_everything.py, line 208 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 209 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 209 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 210 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 211 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 211 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 212 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 212 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 213 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 213 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 213 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 214 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 214 - E231 missing whitespace after ':'
* corehq/apps/reminders/tests/test_everything.py, line 214 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 215 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 216 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 217 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 218 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 218 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 219 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 219 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 219 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 220 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 220 - E231 missing whitespace after ':'
* corehq/apps/reminders/tests/test_everything.py, line 220 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 221 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 227 - E501 line too long (116 > 115 characters)
* corehq/apps/reminders/tests/test_everything.py, line 240 - E303 too many blank lines (2)
* corehq/apps/reminders/tests/test_everything.py, line 361 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 363 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 363 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 364 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 364 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 364 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 364 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 365 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 365 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 365 - E231 missing whitespace after ':'
* corehq/apps/reminders/tests/test_everything.py, line 365 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 366 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 366 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 367 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 368 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 368 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 369 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 369 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 370 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 370 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 370 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 371 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 371 - E231 missing whitespace after ':'
* corehq/apps/reminders/tests/test_everything.py, line 371 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 372 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 378 - E501 line too long (116 > 115 characters)
* corehq/apps/reminders/tests/test_everything.py, line 379 - E225 missing whitespace around operator
* corehq/apps/reminders/tests/test_everything.py, line 430 - E501 line too long (150 > 115 characters)
* corehq/apps/reminders/tests/test_everything.py, line 437 - E221 multiple spaces before operator
* corehq/apps/reminders/tests/test_everything.py, line 437 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 438 - E221 multiple spaces before operator
* corehq/apps/reminders/tests/test_everything.py, line 438 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 439 - E221 multiple spaces before operator
* corehq/apps/reminders/tests/test_everything.py, line 439 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 440 - E221 multiple spaces before operator
* corehq/apps/reminders/tests/test_everything.py, line 440 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 441 - E221 multiple spaces before operator
* corehq/apps/reminders/tests/test_everything.py, line 441 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 466 - E501 line too long (150 > 115 characters)
* corehq/apps/reminders/tests/test_everything.py, line 493 - E501 line too long (150 > 115 characters)
* corehq/apps/reminders/tests/test_everything.py, line 509 - E501 line too long (150 > 115 characters)
* corehq/apps/reminders/tests/test_everything.py, line 525 - E501 line too long (150 > 115 characters)
* corehq/apps/reminders/tests/test_everything.py, line 552 - E501 line too long (150 > 115 characters)
* corehq/apps/reminders/tests/test_everything.py, line 568 - E501 line too long (150 > 115 characters)
* corehq/apps/reminders/tests/test_everything.py, line 575 - E221 multiple spaces before operator
* corehq/apps/reminders/tests/test_everything.py, line 575 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 576 - E221 multiple spaces before operator
* corehq/apps/reminders/tests/test_everything.py, line 576 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 577 - E221 multiple spaces before operator
* corehq/apps/reminders/tests/test_everything.py, line 577 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 578 - E221 multiple spaces before operator
* corehq/apps/reminders/tests/test_everything.py, line 578 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 579 - E221 multiple spaces before operator
* corehq/apps/reminders/tests/test_everything.py, line 579 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 594 - E501 line too long (150 > 115 characters)
* corehq/apps/reminders/tests/test_everything.py, line 605 - E501 line too long (116 > 115 characters)
* corehq/apps/reminders/tests/test_everything.py, line 621 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 623 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 623 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 624 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 624 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 624 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 624 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 625 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 625 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 625 - E231 missing whitespace after ':'
* corehq/apps/reminders/tests/test_everything.py, line 625 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 626 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 626 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 646 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 648 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 648 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 649 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 649 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 649 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 649 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 650 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 650 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 650 - E231 missing whitespace after ':'
* corehq/apps/reminders/tests/test_everything.py, line 650 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 651 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 651 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 671 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 673 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 673 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 674 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 674 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 674 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 674 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 675 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 675 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 675 - E231 missing whitespace after ':'
* corehq/apps/reminders/tests/test_everything.py, line 675 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 676 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 676 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 719 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 720 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 720 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 723 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 724 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 724 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 745 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 746 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 746 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 749 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 750 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 750 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 768 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 769 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 769 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 772 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 773 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 773 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 777 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 778 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 778 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 781 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 782 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 782 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 800 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 801 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 801 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 804 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 805 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 805 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 808 - E302 expected 2 blank lines, found 1
* corehq/apps/reminders/tests/test_everything.py, line 813 - E501 line too long (116 > 115 characters)
* corehq/apps/reminders/tests/test_everything.py, line 829 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 831 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 831 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 832 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 832 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 832 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 832 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 833 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 833 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 833 - E231 missing whitespace after ':'
* corehq/apps/reminders/tests/test_everything.py, line 833 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 834 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 834 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 852 - E303 too many blank lines (2)
* corehq/apps/reminders/tests/test_everything.py, line 867 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 868 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 868 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 891 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 899 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 900 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 900 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 905 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 913 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 914 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 914 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 939 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 947 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 948 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 948 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 953 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 961 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 962 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 962 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 994 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 995 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 995 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 1007 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 1008 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 1008 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 1029 - E302 expected 2 blank lines, found 1
* corehq/apps/reminders/tests/test_everything.py, line 1034 - E501 line too long (116 > 115 characters)
* corehq/apps/reminders/tests/test_everything.py, line 1050 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 1052 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 1052 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 1053 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 1053 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 1053 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 1053 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 1054 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 1054 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 1054 - E231 missing whitespace after ':'
* corehq/apps/reminders/tests/test_everything.py, line 1054 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 1055 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 1055 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 1073 - E303 too many blank lines (2)
* corehq/apps/reminders/tests/test_everything.py, line 1086 - E203 whitespace before ','
* corehq/apps/reminders/tests/test_everything.py, line 1087 - E131 continuation line unaligned for hanging indent
* corehq/apps/reminders/tests/test_everything.py, line 1087 - E231 missing whitespace after ','
* corehq/apps/reminders/tests/test_everything.py, line 1108 - E251 unexpected spaces around keyword / parameter equals
* corehq/apps/reminders/tests/test_everything.py, line 1108 - E501 line too long (121 > 115 characters)

**Comment by dannyroberts** [comment]
@czue builds off https://github.com/dimagi/commcare-hq/pull/8534

**Comment by dannyroberts** [comment]
Only the last 3 commits are new, the rest are from the other PR

**Comment by gcapalbo** [comment]
@dannyroberts see my comment here: https://github.com/dimagi/commcare-hq/pull/8534#issuecomment-146243538

**Comment by czue** [comment]
(will wait for you and @gcapalbo to sort out next steps before review)

File: corehq/couchapps/all_docs/views/by_domain_doc_type/map.js (hunk 1 above)
Comment (czue) [inline corehq/couchapps/all_docs/views/by_domain_doc_type/map.js]: will be nice to have this finally

**Comment by czue** [comment]
:+1: to merge whenever you're good

**Comment by gcapalbo** [comment]
good to merge from my side

**Comment by dannyroberts** [comment]
Net change: -1 view
Indexing load: 10 views in `_design/domain`, 2 views in `_design/all_docs`

File: corehq/couchapps/all_docs/views/by_domain_doc_type/map.js (hunk 1 above)
Comment (dimagimon) [inline corehq/couchapps/all_docs/views/by_domain_doc_type/map.js]: Missing name in function declaration.

**Comment by dannyroberts** [comment]
Net change: -2 view
Indexing load: 9 views in `_design/domain`, 2 views in `_design/all_docs`

**Comment by dannyroberts** [comment]
@esoergel I think your domain views cleanup included this and more, so closing

**Pull request closed without merging**